src/cli-ng/abrtcli/cli.py

```diff
@@ -246,10 +245,10 @@ def retrace(args):
 
         if remote:
             print(_('Remote retracing'))
-            run_event('analyze_RetraceServer', prob)
+            libreport.run_event_on_problem_dir(prob.path, 'analyze_RetraceServer')
```

Comment:
`run_event` was always logging an error message but `run_event_on_problem_dir` logs errors only if you pass True as the 3rd argument.